custom/pagination/components/PaginatedVideoGrid.js

```diff
@@ -219,75 +232,82 @@ export const PaginatedVideoGrid = ({
       layer = 0;
     }
 
-    const receiveSettings = visibleParticipants.reduce(
-      (settings, participant) => {
-        if (isLocalId(participant.id)) return settings;
-        return { ...settings, [participant.id]: { video: { layer } } };
+    const receiveSettings = visibleParticipantIds.reduce(
+      (settings, participantId) => {
+        if (localParticipant.session_id === participantId) return settings;
+        return { ...settings, [participantId]: { video: { layer } } };
       },
       {}
     );
     callObject.updateReceiveSettings(receiveSettings);
-  }, [visibleParticipants, callObject]);
+  }, [visibleParticipantIds, callObject, localParticipant.session_id]);
 
   // -- Active speaker
 
   /**
    * Handle position updates based on active speaker events
    */
-  const handleActiveSpeakerChange = useCallback(
-    (peerId) => {
-      if (!peerId) return;
-      // active participant is already visible
-      if (visibleParticipants.some(({ id }) => id === peerId)) return;
-      // ignore repositioning when viewing page > 1
-      if (page > 1) return;
-
-      /**
-       * We can now assume that
-       * a) the user is looking at page 1
-       * b) the most recent active participant is not visible on page 1
-       * c) we'll have to promote the most recent participant's position to page 1
-       *
-       * To achieve that, we'll have to
-       * - find the least recent active participant on page 1
-       * - swap least & most recent active participant's position via setParticipantPosition
-       */
-      const sortedVisibleRemoteParticipants = visibleParticipants
-        .filter(({ isLocal }) => !isLocal)
-        .sort((a, b) => sortByKey(a, b, 'lastActiveDate'));
-
-      if (!sortedVisibleRemoteParticipants.length) return;
-
-      swapParticipantPosition(sortedVisibleRemoteParticipants[0].id, peerId);
-    },
-    [page, swapParticipantPosition, visibleParticipants]
+  const sortedVisibleRemoteParticipantIds = useDeepCompareMemo(
```

Comment:
I am guessing it doesn't make sense to use/showcase the hooks `useParticipantIds()` `sort` parameter here?